### DISCUSSION TITLE
Is it possible to disable legacy cookies?

### DISCUSSION THREAD

**Discussion by rome-user** [keycloak/keycloak#16710] [Q&A] [answered]
The legacy cookies are not needed for my use case (we target desktop only and we do not expect to support old Safari versions). Every time my front-end application pings a token endpoint, Chrome dev tools will complain about the lack of SameSite attributes. So I would like to know whether we can disable Keycloak from sending legacy cookies. Thank you.

**Comment by jonkoops** [comment, marked as answer]
We're actually in the process of possibly removing the legacy cookies entirely, for more information see https://github.com/keycloak/keycloak/issues/16181.

**Reply by krishnakumarakurathi** [reply]
Dear @jonkoops,

Has this been addressed? If so, in which version? If not, could you provide a rough ETA or indicate the version in which it will be fixed?

**Reply by jonkoops** [reply]
You can see the status on the issue I linked https://github.com/keycloak/keycloak/issues/16181#issuecomment-2114599025. No I cannot provide an ETA.
